engine/src/main/scala/cromwell/services/ServiceRegistryActor.scala

```diff
@@ -0,0 +1,51 @@
+package cromwell.services
+
+import akka.actor.{Actor, ActorRef, Props}
+import com.typesafe.config.{ConfigFactory, Config, ConfigObject}
+import cromwell.engine.db.DataAccess
+import cromwell.services.ServiceRegistryActor._
+import lenthall.config.ScalaConfig._
+
+import scala.collection.JavaConverters._
+
+object ServiceRegistryActor {
+  case class ServiceRegistryFailure(serviceName: String)
+  trait ServiceRegistryMessage {
+    def serviceName: String
+  }
+
+  def props(dataAccess: DataAccess, config: Config) = {
+    Props(ServiceRegistryActor(dataAccess, config))
+  }
+}
+
+case class ServiceRegistryActor(dataAccess: DataAccess, globalConfig: Config) extends Actor {
```

Comment:
IMHO, this class should not be getting a DataAccess as a parameter. I see that it's required by the KV Service actor in the constructor, but when we start adding other service actor implementations, we will begin to notice a discrete seemingly unrelated params being passed onto here as a constructor. 

I feel this actor should receive a `Seq[Props]` (or a Seq of some ADT with props) as a param, and all it should do is create the 'services` map of String -> ActorRef using those props. The responsibility of reading the config and creating the props object can can be delegated to something like a Factory, which will create these props and know the args required by the services.